tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/intersection/IntersectionTypeTest.java

```diff
@@ -106,6 +106,11 @@ public void testIntersectionAsFieldInAnonymousRecord() {
         BRunUtil.invoke(errorIntersectionResults, "testIntersectionAsFieldInAnonymousRecord");
     }
 
+    @Test
+    public void testIntersectionOfErrorWithInlineError() {
+        BRunUtil.invoke(errorIntersectionResults, "testIntersectionOfErrorWithInlineError");
```

Comment:
Shall we also add some negative tests for the errors which we couldn't define previously?

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/28595/commits/0ebd815eccbc9d5ff2c69a7c9ef22d2e5a131db4